test/fixtures/lightscript/for/indexless-syntax/actual.js

```diff
@@ -0,0 +1 @@
+for 0 til 3:
```

Comment:
I put that in to test the parser crash, but I guess now it's cleared up we don't need that. Removing.